### PR TITLE
Fix Standalone dapr init behavior.

### PR DESCRIPTION
**Pull request by mukundansundar**
Remove uninstall all from init path. Add install path arg to uninstall cmd. Cleanup binaries, archive correctly.

# Description

Removed `uninstall(true)` from standalone init path. 
Removed archive, binary from default `.dapr` folder. 
Remove binary from default installation path or custom installation path specified by `--install-path` arg. 
Do not create/run `dapr_zipkin`, `dapr_redis` containers if already exists rather start it if already exists. 
Do not initialize dapr again if `daprd` is installed or `dapr_palcement` exists/running.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #380 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [x] Extended the documentation

### CODE REVIEW

**Comment by youngbupark** [comment]
@mukundansundar we have one liner  install script. I think we also need to provide the option to select `dapr cli` path. 

https://github.com/dapr/cli/tree/master/install

Can you please create the separate issue ?

File: README.md [hunk 1]
```diff
@@ -56,16 +56,32 @@ Each release of Dapr CLI includes various OSes and architectures. These binary v
...
 ⌛  Making the jump to hyperspace...
-↗   Downloading binaries and setting up components...
-✅  Success! Dapr is up and running
+↓  Downloading binaries and setting up components...
+removing archive /Users/msundar/.dapr/daprd_darwin_amd64.tar.gz
+↙  Downloading binaries and setting up components...
+installing Dapr to /usr/local/bin
+
+removing extracted binary /Users/msundar/.dapr/daprd
+✅  Downloading binaries and setting up components...
```
Comment (amanbha) [inline README.md:65]: Change the text to something like (in the code as well): downloaded and configured

File: README.md [hunk 2]
```diff
@@ -120,6 +135,14 @@ $ dapr uninstall --all
 
 You should always run a `dapr uninstall` before running another `dapr init`.	
 
+#### Uninstall Dapr from a specific install path 
+
+If previously installed to a specific location, Dapr can be unsintalled with the `--install-path` argument
+
+```bash
+dapr uninstall --install-path ~/.dapr
```
Comment (amanbha) [inline README.md:143]: for this example, use some path other than ~/.dapr. As ~/.dapr is default folder for components on MAc.

File: README.md [hunk 3]
```diff
@@ -56,16 +56,32 @@ Each release of Dapr CLI includes various OSes and architectures. These binary v
...
+✅  Success! Dapr is up and running. To get started, go here: https://aka.ms/dapr-getting-started
 ```
 
 > Note: To see that Dapr has been installed successfully, from a command prompt run the `docker ps` command and check that the `daprio/dapr:latest`,  `dapr_redis` and `dapr_zipkin` container images are all running.
 
 This step creates the following defaults:
-1. components folder which is later used at runtime unless the --components-path option is provided. For Linux/MacOS, the default components folder path is $HOME/.dapr/components and for Windows it is %USERPROFILE%\.dapr\components.
-2. component files in the components folder called pubsub.yaml, statestore.yaml and zipkin.yaml. 
-3. default config file $HOME/.dapr/config.yaml for Linux/MacOS or for Windows at %USERPROFILE%\.dapr\config.yaml to enable tracing on `dapr init` call. Can be overridden with the `--config` flag on `dapr run`.
+1. components folder which is later used at runtime unless the `--components-path` option is provided. For Linux/MacOS, the default components folder path is `$HOME/.dapr/components` and for Windows it is `%USERPROFILE%\.dapr\components`.
```
Comment (amanbha) [inline README.md:82]: it should read "components folder which is later used later with dapr run"

**Pull request merged**